### PR TITLE
feat: Improve curation explorer

### PR DESCRIPTION
**Pull request by FedeMadoery**
### First needs to be reviewed and merged PR #54

# Description

- Badge Evidence Displayer for curation list
- Improve how we get the evidence value
- Improve evidence display elements

### CODE REVIEW

File: i18n/translations.en.json [hunk 1]
```diff
@@ -65,12 +65,19 @@
         "showOthers": "See minted badges"
       },
       "creator": {
-        "createdBy": "Created By"
+        "createdBy": "Crated By"
```
Comment (Agupane) [inline i18n/translations.en.json:68]: typo

File: pages/badge/curate.tsx [hunk 2]
```diff
@@ -1,46 +1,97 @@
...
+        behavior: 'smooth',
+      })
+    }
+  }, [badgesElementRefs, selectedBadge])
+
+  useEffect(() => {
+    if (badges.length && rightPress) {
+      selectNextBadge()
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (Agupane) [inline pages/badge/curate.tsx:73]: why this?, badges should be also part of the dependencies array

File: src/pagePartials/badge/explorer/ListingCriteriaPreview.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  const [badgeCriteriaUrl, setBadgeCriteriaUrl] = useState<string | undefined>(undefined)
+
+  useEffect(() => {
+    setBadgeCriteriaUrl(undefined)
+  }, [badgeTypeId])
+
+  const badgeTypeData = useBadgeType(badgeTypeId)
+  const badgeTypeMetadata = badgeTypeData.data?.badgeTypeMetadata
+
+  if (!badgeTypeMetadata) {
```
Comment (Agupane) [inline src/pagePartials/badge/explorer/ListingCriteriaPreview.tsx:19]: this breaks the rules of hooks, its not a good practice to use conditionals outside the hooks, you should put this if inside the second useEffect

**Review by Agupane — CHANGES_REQUESTED** [review]

File: pages/badge/curate.tsx (hunk 2 above)
Comment (FedeMadoery) [reply pages/badge/curate.tsx:73]: I ask the same, haha, why the badges should be part of it? Let's continue this conversation on Discord.

File: src/pagePartials/badge/explorer/ListingCriteriaPreview.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+import { Box, IconButton } from '@mui/material'
+
+import PDFViewer from '@/src/components/common/PDFViewer'
+import useBadgeType from '@/src/hooks/subgraph/useBadgeType'
+
+export function ListingCriteriaPreview({ badgeTypeId }: { badgeTypeId: string }) {
+  const [badgeCriteriaUrl, setBadgeCriteriaUrl] = useState<string | undefined>(undefined)
+
+  useEffect(() => {
+    setBadgeCriteriaUrl(undefined)
```
[the remark this answers is not in the document]
Comment (FedeMadoery) [reply src/pagePartials/badge/explorer/ListingCriteriaPreview.tsx:13]: Good catch, I need to remove it, I was trying something with the previous lib.

File: src/pagePartials/badge/explorer/ListingCriteriaPreview.tsx (hunk 3 above)
Comment (FedeMadoery) [reply src/pagePartials/badge/explorer/ListingCriteriaPreview.tsx:19]: You are half right haha... Let's continue on discord.

**Pull request merged**